### DISCUSSION TITLE
workflow/cli/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/workflow#28] [General]
# workflow/cli/

https://shanenull.com/workflow/cli/

**Comment by shane0** [comment]
- https://shanenull.com/workflow/click/
- https://shanenull.com/workflow/cookiecutter/
- https://shanenull.com/workflow/todo/

**Reply by shane0** [reply]
https://shanenull.com/workflow/alias/

**Reply by shane0** [reply]
https://shanenull.com/workflow/alias/
